react/src/lib/components/DeckGLMap/components/settings/LayerSettingsButton.test.tsx

```diff
@@ -6,14 +6,12 @@ import { testStore, Wrapper } from "../../test/TestWrapper";
 import LayerSettingsButton from "./LayerSettingsButton";
 
 describe("test layers settings button", () => {
+    // render function is rendering just 'null'
     it("snapshot test", () => {
         const { container } = render(
             Wrapper({
                 children: (
-                    <LayerSettingsButton
-                        layerId="colormap-layer"
-                        layerType="ColormapLayer"
-                    />
+                    <LayerSettingsButton layerId="" layerType="ColormapLayer" />
```

Comment:
The `layerId` prop needs to match the ColormapLayer `id` field from deckgl-map.json. Now it is `"colormap-layer"`.
Same for the rest of LayerSettingsButton components in this file.